packages/react-components/src/components/VideoTile.tsx

```diff
@@ -295,6 +302,19 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
                   <Icon iconName="VideoTileMicOff" />
                 </Stack>
               )}
+              {
+                /* @conditional-compile-remove(pinned-participants) */
+                isPinned && (
+                  <Stack className={mergeStyles(iconContainerStyle)}>
+                    <Icon
+                      iconName="VideoTilePinned"
+                      className={mergeStyles({
```

Comment:
Let's move this style to the VideoTile.styles file

Reply:
done